### DISCUSSION TITLE
Welcome Registration page

### DISCUSSION THREAD

**Discussion by joomleb** [joomla/joomla-cms#37966] [General]
Hi guys, 
we are immersed in an Engage World,  a simple Welcome Registration page is a must, as we have yet for the Login process.
This is why I think that a Registration Form menu item redirection as described here: https://github.com/joomla/joomla-cms/issues/37566 is really a "big" missing.

Are you agree ?
Please, Is there any hope of seeing it added to the 4.2 release ?

**Comment by grant32** [comment]
Although I'm not a support member, but hopefully it will be added to 4.2. [pkw reinigung](https://www.autoreinigung-noack.de/)

**Comment by brianteeman** [comment]
There is nothing that needs to be done as it is already available.

On completing a registration  you are redirected to a view (complete.php). Just create a template override of that view with the content that you desire
